core/kazoo_number_manager/src/carriers/knm_bandwidth2.erl

```diff
@@ -116,7 +116,7 @@ query_local_bw_numbers(Number, Quantity, Options, AccountId)
     ViewOptions = [{'startkey', [?NUMBER_STATE_DISCOVERY, BW2Module, Number]}
                   ,{'endkey', [?NUMBER_STATE_DISCOVERY, BW2Module, <<"\ufff0">>]}
                   ,{'limit', Quantity}
-                  ,{skip, Offset}
+                  ,{skip, Offset+1}
```

Comment:
@fenollp maybe the problem is with the providers implementing the find_more logic ? or i'm missing something, can't see why knm_managed would loop.